scripts/search-job-messages-trend-antimalware.py

```diff
@@ -0,0 +1,48 @@
+# Submits search job, waits for completion, then prints and emails _messages_
```

Comment:
This script seems specific, so would be great to explain what it does more explicitly.